Composer/packages/client/src/store/action/dialog.ts

```diff
@@ -112,20 +114,25 @@ export const debouncedUpdateDialog = debounce(async (store, id, content) => {
   }
 }, 500);
 
-export const updateDialogBase: ActionCreator = (store, { id, content }) => {
+export const updateDialogBase: ActionCreator = (store, { id, content }, externalUpdate?: ExternalUpdate) => {
```

Comment:
So the design is when some content is updated in shell, we mark the "externalUpdate" to notify any component (like Form) which may holds an local data to refresh, right

Reply:
I think we just need to give a better name for this